code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1152,13 +1153,13 @@
 	if((temp_turf.z != our_turf.z) || M.stat!=CONSCIOUS) //Not on the same zlevel as us or they're dead.
 //		to_chat(world, "[(temp_turf.z != our_turf.z) ? "not on the same zlevel as [M]" : "[M] is not concious"]")
 		if(temp_turf.z != 2)
-			to_chat(src, "The mind of [M] is too faint...")//Prevent "The mind of Admin is too faint..."
-
-
+			if(prob(5))//Can't tell who is missing as easily as before. I feel like this is a fun alternative to removing it altogether
```

Comment:
I'd also like to point out that psy resist is under the stealth section of
goon powers. So I'm trying to keep it true to its original intent. You
shouldn't know if someone is hiding from you. You should be suspicious. And
that's why you can rapidly ping for resisting entities if you get
suspicious.

I really do feel like there's something I'm not understanding here...